assets/bindata_version_hash.go

```diff
@@ -0,0 +1,5 @@
+package assets
```

Comment:
Can we add a comment here to say that this shouldn't be edited?

Reply:
✅ done